File: python/7 kyu/106.py
# https://www.codewars.com/kata/58ddffda929dfc2cae0000a5/train/python

# Passed

def clonewars(kata_per_day):
    total_clones = 1
    total_kata_solved = 0
    for solutions_for_day in range(kata_per_day, 0, -1):
        total_kata_solved += total_clones * solutions_for_day
        total_clones *= 2 if solutions_for_day != 1 else 1

    result = [total_clones, total_kata_solved]
    return result

output = clonewars(5)
print(output)

File: nodejs/7 kyu/106.js
// https://www.codewars.com/kata/59c62f1bdcc40560a2000060/train/javascript

// Passed

function solve(a) {
  let oddCount = 0;
  let evenCount = 0;
  for (const elem of a) {
    if (typeof elem === "number") {
      if (elem % 2 === 0) {
        evenCount++;
      } else {
        oddCount++;
      }
    }
  }

  let evenOddDisparity = evenCount - oddCount;
  return evenOddDisparity;
}

const output = solve([
  5,
  15,
  16,
  10,
  6,
  4,
  16,
  "t",
  13,
  "n",
  14,
  "k",
  "n",
  0,
  "q",
  "d",
  7,
  9,
]);
console.log(output);